matrix_reminder_bot/bot_commands.py

```diff
@@ -338,8 +338,25 @@ async def _remind_me(self):
         """Set a reminder that will remind only the user who created it"""
         await self._remind(target=self.event.sender)
 
+    async def _insufficient_room_ping_pl(self):
+        room_ping_pl = self.room.power_levels.defaults.notifications.get('room')
+        sender_pl = self.room.power_levels.users.get(self.event.sender, 0)
+        bot_pl = self.room.power_levels.users.get(self.client.user_id, 0)
+        if sender_pl < room_ping_pl:
+            insufficient = f"You (<code>{sender_pl}</code>) do"
+        elif bot_pl < room_ping_pl:
```

Comment:
the use case of #80 is not possible to partially work around with PLs anymore, so it now needs to get addressed more than ever.